tsdb/agent/db_test.go

```diff
@@ -138,6 +148,26 @@ func TestCommit(t *testing.T) {
 		}
 	}
 
+	lbls = labelsForTest(t.Name()+"_histogram", numSeries)
```

Comment:
Instead of the same testHistogram() and testFloatHistogram(), we can re-use these histogram generators from `tsdb` package.
https://github.com/prometheus/prometheus/blob/d82ea2eb1c4bd51ca70d181e358215d1023d8cf7/tsdb/head.go#L2027-L2125